docs/components/processor.rst

```diff
@@ -47,6 +47,31 @@ crash id to the "socorro.normal" RabbitMQ queue.
 We have helper scripts for these steps.
 
 
+``scripts/fetch_crashids.py``
+-----------------------------
+
+This will generate a list of crash ids from -prod that meet specified criteria.
+Crash ids are printed to stdout, so you can use this in conjunction with other
+scripts or redirect to a file.
+
+This pulls 100 crash ids from yesterday for Firefox product::
+
+  $ docker/as_me.sh scripts/fetch_crashids.py
+
+This pulls 5 crash ids from 2017-09-01::
+
+  $ docker/as_me.sh scripts/fetch_crashids.py --num=5 --date=2017-09-01
+
+This pulls 100 crash ids for criteria specified with a Super Search url that we
+copy and pasted::
+
+  $ docker/as_me.sh scripts/fetch_crashids.py "--url=https://crash-stats.mozilla.com/search/?product=Firefox&date=%3E%3D2017-09-05T15%3A09%3A00.000Z&date=%3C2017-09-12T15%3A09%3A00.000Z&_sort=-date&_facets=signature&_columns=date&_columns=signature&_columns=product&_columns=version&_columns=build_id&_columns=platform#facet-signature"
```

Comment:
This line is probably going to look terrible in the docs, but I wasn't sure what else to do since Super Search urls are long.